.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,7 +27,7 @@
 - [ ] Logging code meets the guideline (TODO)
 - [ ] OpenAPI and Postman files updated, client regenerated (`make generate-client`) (if needed)
 - [ ] User guide and development docs updated (if needed)
-
+- [ ] Related Helm chart(s) updated (if needed)
```

Comment:
```suggestion
- [ ] Helm chart and config examples updated (if needed)

```

Reply:
We deploy several charts in the posthooks by related charts i meant mostly those.

Reply:
We can write detailed explanation in the contribution guideline for these checklist elements. Their purpose is to remind the author of some important things so that they are not missed.

IMHO `Related Helm chart(s) updated` is good enough for posthook installed charts as well.